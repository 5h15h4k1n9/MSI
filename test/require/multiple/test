nestedRequire(150)
multiEffect(3, 10)
counter
flag
exit
