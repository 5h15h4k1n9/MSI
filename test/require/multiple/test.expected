Reverted with: Too big
(void) void
(int) 7
(bool) true
Bye-bye!
